### DISCUSSION TITLE
VimExchange extension highlighting issue

### DISCUSSION THREAD

**Discussion by tommcdo** [JetBrains/ideavim#388] [General]
Hey IdeaVim community!

I'm the author of the original (native Vim) [exchange plugin](https://github.com/tommcdo/vim-exchange). A user opened an issue about line highlighting. Since the issue doesn't exist in the native Vim plugin, I'm guessing there might be a bug in the IdeaVim port of it.

https://github.com/tommcdo/vim-exchange/issues/57

Let's start the discussion here and see what we can figure out!

**Comment by AlexPl292** [comment]
Hi, thank you for the report. The issue will be fixed in the coming release. https://youtrack.jetbrains.com/issue/VIM-2359

And thank you for the great plugin for vim :D
